lib/code_fund/campaigns/campaigns.ex

```diff
@@ -144,6 +144,14 @@ defmodule CodeFund.Campaigns do
     Repo.delete(campaign)
   end
 
+  def get_active_campaign_for_property_with_biggest_bid_amount(%Property{} = property) do
+    get_active_campaign_for_property(property) |> order_by(desc: :bid_amount) |> Repo.one()
+  end
+
+  def get_random_active_campaign_for_property(%Property{} = property) do
+    get_active_campaign_for_property(property) |> order_by(fragment("RANDOM()")) |> Repo.one()
```

Comment:
Does this still only return the highest bid? This seems to return any bid..

I just want to make sure it returns the random one of the highest matching bidders.